scylla/src/frame/cql_types_test.rs

```diff
@@ -25,15 +26,15 @@ async fn test_cql_types() {
         .await
         .unwrap();
 
-    let date_ts: u32 = 10;
+    let date = NaiveDate::from_ymd(2020, 1, 1);
     session
-        .query("INSERT INTO ks.days (day, id) VALUES (?, 1)", (date_ts,))
+        .query("INSERT INTO ks.days (day, id) VALUES (?, 1)", (date,))
         .await
         .unwrap();
 
     if let Some(rows) = session.query("SELECT day FROM ks.days", &[]).await.unwrap() {
-        for row in rows.into_typed::<(u32,)>() {
-            let day: u32 = row.unwrap().0;
+        for row in rows.into_typed::<(NaiveDate,)>() {
+            let day: NaiveDate = row.unwrap().0;
             println!("day: {}", day);
```

Comment:
Looks good, but please also assert that the value is what's expected (1.1.2020).